gauntlet/packages/gauntlet-solana-contracts/artifacts/schemas/serum_multisig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.0.0",
+  "version": "0.8.0",
```

Comment:
This one should be `"version": "0.9.0"`. You can take it from the multisig PR #157.